src/ansys/simai/core/data/models.py

```diff
@@ -120,3 +233,24 @@ def build(
                 dismiss_data_with_volume_overflow,
             )
         )
+
+    def doa_axis_definition(
```

Comment:
@tmpbeing I saw your comment above:
> I think not raising a ValidationError would be nice (use custom errors wherever we can)

But IMO writing functions just to try/except like this is bloat, maybe we can make `InvalidArguments` subclass `ValidationError` ? Or just reexport it as `ansys.simai.core.ValidationError` ?

Reply:
`pydantic.ValidationError` is final and it cannot be subclassed. Maybe not using pydantic overall would be a solution :/

Reply:
the method `doa_axis_definition` is removed and the current implementation does not uses pydantic features. `InvalidArguments` exceptions are raised in cases of erroneous parameters of `DomainAxisDefinition`.